baseclasses/solvers/pyAero_solver.py

```diff
@@ -103,10 +114,11 @@ def setDVGeo(self, DVGeo, pointSetKwargs=None):
 
         self.DVGeo = DVGeo
 
-        if pointSetKwargs is None:
-            self.pointSetKwargs = {}
-        else:
-            self.pointSetKwargs = pointSetKwargs
```

Comment:
This code avoids [mutable defaults](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/), so we should keep it like this.